lib/pfconfig/manager.pm

```diff
@@ -197,26 +200,18 @@ That sends the signal that the raw memory is expired
 
 sub touch_cache {
     my ( $self, $what ) = @_;
-    my $logger = pfconfig::log::get_logger;
     $what =~ s/\//;/g;
     my $filename = pfconfig::util::control_file_path($what);
     $filename = untaint_chain($filename);
 
-    if ( !-e $filename ) {
-        my $fh;
-        unless ( open( $fh, ">$filename" ) ) {
-            $logger->error("Can't create $filename\nPlease run 'pfcmd fixpermissions'");
-            return 0;
-        }
+    if (sysopen(my $fh,$filename,O_RDWR | O_CREAT)) {
+        POSIX::2008::futimens(fileno $fh);
```

Comment:
Ok I normalized POSIX::2008::futimens to use the same resolution of Time::HiRes.
My changes have been pushed please re-review